OSDOCS-9612: Added instructions for the additional security groups tag for ROSA with HCP clusters (#72838)
openshift/openshift-docs: modules/creating-a-machine-pool-cli.adoc

* A Single-AZ machine pool with one availability zone can have a machine count in multiples of 1, such as 1,2,3,4 and so on.
====
<10> Optional: For machine pools in clusters that do not have Red Hat managed VPCs, you can select additional custom security groups to use in your machine pools. You must have already created the security groups and associated them with the VPC that you selected for this cluster. You cannot add or edit security groups after you create the machine pool. For more information, see the requirements for security groups in the "Additional resources" section.
+
[IMPORTANT]
====

> You can only use ten additional security groups for machine pools on {hcp-title} clusters. 

Reviewer: At most ten.  The highest number is limited by customer's network interface quota. Otherwise customer will encounter below error
`Not enough quota for Security Groups for '%s'." +
		" There must be a minimum of '%d' quota for Security Groups per Network Interface." +
		" Please adjust your AWS quota for code 'L-2AFB9258'
`

Author: This work? I tried to write more of an explanation, but then, I felt like I was overwhelming the users. Should we explain why this limit exists?

![image](https://github.com/openshift/openshift-docs/assets/16167833/d09b5635-8206-4e77-b70a-7d4f53902e8f)

> You can only use, at most, ten additional security groups for machine pools on {hcp-title} clusters. 

Reviewer 2: ```suggestion
You can up to ten additional security groups for machine pools on {hcp-title} clusters. 
```
